src/Symfony/Component/Runtime/Runner/Middleware/MiddlewareFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime\Runner\Middleware;
+
+/**
+ * @author Sascha Heilmeier<sascha.heilmeier@netlogix.de>
+ */
+readonly class MiddlewareFactory
+{
+    /**
+     * @param class-string<MiddlewareInterface> ...$middleware
```

Comment:
```suggestion
     * @param class-string<MiddlewareInterface> $middleware
```
Unsure, but I think we don't need the three dots here